src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java

```diff
@@ -203,6 +237,7 @@ public FormValidation doCheckReturnStatus(@QueryParameter boolean returnStdout,
             if (!directory) {
                 throw new AbortException("missing workspace " + remote + " on " + node);
             }
+            LOGGER.log(Level.FINE, "{0} seems to be online", node);
```

Comment:
DEBUG/FINEST? maybe also makes sense to print Execution#toString() to the log so we understand from where it comes

Reply:
Sure.

Reply:
(#63)